homework/home3.2/src/Triangle.java

```diff
@@ -0,0 +1,41 @@
+package org.example;
+
+public class Triangle {
+    private double a, b, c;
+
+    public Triangle(double a, double b, double c) {
+        this.a = a;
+        this.b = b;
+        this.c = c;
+    }
+
+    public double area() {
+        if (a <= 0 || b <= 0 || c <= 0) {
+            return -1; // Indicate invalid triangle
+        }
+        if (a + b <= c || a + c <= b || b + c <= a) {
+            return -1; // Indicate invalid triangle
+        }
+
+        double s = (a + b + c) / 2.0;
+        return Math.sqrt(s * (s - a) * (s - b) * (s - c));
+    }
+
+    public double getHeight(char side) {
```

Comment:
use enum to move user input conversion from responsibility area of the class
and of course optional double instead of magic number -1